packages/victory-core/src/types/callbacks.ts

```diff
@@ -33,13 +34,19 @@ export type StringOrNumberOrCallback =
   | number
   | VictoryStringOrNumberCallback;
 export type StringOrCallback = string | VictoryStringCallback;
-
 export type SliceNumberOrCallback<
   T,
   P extends string | number | symbol = never
 > = number | ((props: Omit<T, P>) => number);
-
-export type CoordinatesPropType = {
-  x: number;
-  y: number;
-};
+export type VictoryNumberCallback = (args: CallbackArgs) => number;
+export type VictoryPaddingCallback = (
+  args: CallbackArgs
+) => number | BlockProps;
+export type VictoryOrientationCallback = (
+  args: CallbackArgs
+) => OrientationTypes;
+export type NumberOrCallback = number | VictoryNumberCallback;
+export type PaddingOrCallback = number | BlockProps | VictoryPaddingCallback;
+export type OrientationOrCallback =
+  | OrientationTypes
+  | VictoryOrientationCallback;
```

Comment:
Just about all of the types in this PR were shuffled around from various files.  Few were changed.